spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+describe ReviewsController do
+  describe "POST create" do
+    context "if user not logged in" do
+      it "redirects to login page" do
+        session[:user_id] = nil
+        post :create, video_id: Fabricate(:video).id
```

Comment:
I think you don't even need to Fabricate(:video).id here, just put in a random number. I typically use `1` :) it should not even reach `create` action.